examples/plot_newfitting_comparison_oldnew.py

```diff
@@ -0,0 +1,221 @@
+# -*- coding: utf-8 -*-
+"""
+
+================================================================================
+Compare performance between old 1T fitting example and new fitting module
+================================================================================
+
+RADIS has its own fitting feature, as shown in 1 temperature fit example, where you have
+to manually create the spectrum model, input the experimental spectrum and other ground-truths
+into numerous RADIS native functions, as well as adjust the fitting pipeline yourself.
+
+Now with the new fitting module implemented on September 6 2022, all you have to do is to prepare a
+.spec file containing your experimental spectrum, fill some JSON forms describing the ground-truth
+conditions just like how you fill your medical checkup paper, call the function fit_spectrum() and
+let it do all the work! If you are not satisfied with the result, you can simply adjust the parameters
+in your JSON, then recall the function again, until the results are satisfied.
+
+This is way easier and more convenient than dwelling into RADIS documentation to find out and
+learn how to use the current example, especially for new RADIS users. This new fitting module
+aims to provide an end-to-end fitting experience, with minimum amount of RADIS knowledge needed.
+
+This example will run both fitting methods on an experimental spectrum with the same ground-truth
+conditions, so you can compare their overall performance, including number of loops, fitting time,
+and final residuals between experimental and best-fit spectra, as an indicator of fit accuracy.
+
```

Comment:
Hello

Can we reduce this lengthy text and go straight to the point ? 

For instance, we don't need to know when it was implemented; or to know if it's easier than dwelling into RADIS documnetation (we're already in RADIS documentation!).

Reply:
I also think the example should **not** compare old & new implmeentaiton of fitting. Users just want to fit. They'll copy/paste this example and adapt it to their case. They're not going to fit twice. 

Can you mention the other implementation in the docstrings, just add a refernece there ("You could also use the old fitting procedure, see ... " ) 

The comparison , however, should be done **for us (developers)**. This Pull Request may be the right place. Just share some performance results in the discussion

Reply:
Yes, I will trim that lengthy introduction.

Also, if the user just want to fit, they have the `plot_newfitting_Tgas.py` as well as other 2 examples from Mr. @minouHub and Mr. @CorentinGrimaldi. This comparison example is for comparison between old & new.

Should I remove this old-new example and put up another PR dedicated for it only?

Reply:
In the example here you're fitting another physical case than Minou's one. It's important. Add it.

Reply:
The file you are commenting to, the `plot_newfitting_comparison_oldnew.py`, uses the case of [1-temperature fit example](https://radis.readthedocs.io/en/latest/auto_examples/plot_1T_fit.html#sphx-glr-auto-examples-plot-1t-fit-py).

So now there are only 4 spectra being used:
- A synthetic spectrum, in `plot_newfitting_Tgas.py` and `plot_newfitting_comparison_methods.py`
- Mr. Minou's spectrum, in `plot_newfitting_Tgas-molfrac.py`
- Mr. Corentin's spectrum, in `plot_newfitting_Trot-Tvib-molfrac.py`
- The  [1-temperature fit example](https://radis.readthedocs.io/en/latest/auto_examples/plot_1T_fit.html#sphx-glr-auto-examples-plot-1t-fit-py) spectrum, in `plot_newfitting_comparison_oldnew.py`

Which one do I need to add?

Reply:
Oh I misunderstood your point, my apologies Mr. @erwanp . I will replace the spectrum used in this old-new comparison with the one from Mr. @minouHub's case.